app/components/candidate_interface/referees_review_component.rb

```diff
@@ -59,6 +62,15 @@ def relationship_row(referee)
       }
     end
 
+    def reference_type_row(referee)
+      {
+        key: 'Reference type',
+        value: referee.referee_type ? t("referee_type.#{referee.referee_type}") : '',
```

Comment:
Wondering if it's possible to do:

```suggestion
        value: referee.referee_type.capitalize.dasherize || '',
```
E.g.
```ruby
'school_based'.capitalize.dasherize
=> "School-based"
```

Might remove the need for the translation additions and the enum?

Reply:
I'll keep the enum just to ensure we cannot save an incorrect referee type, but thanks for suggesting `dasherize` I tried to use `humanize` but that just added space, it good to know this! 😄